plugin_config.txt

```diff
@@ -1621,6 +1621,18 @@ my $VEP_PLUGIN_CONFIG = {
       "plugin_url" => "https://raw.githubusercontent.com/Ensembl/VEP_plugins/release/108/StructuralVariantOverlap.pm"
     },
 
+    # UTRAnnotator
+    # https://github.com/ensembl-variation/VEP_plugins/blob/main/UTRAnnotator.pm
+    {
+      "key" => "UTRAnnotator",
+      "label" => "UTRAnnotator",
+      "available" => 0,
+      "enabled" => 0,
+      "section" => "Structural variant data",
```

Comment:
I don't think structural variant is the most appropriate, the plugin annotates small variants (1-5bp).
As the output is transcript specific it could be under "Transcript annotation".